integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonLateDecodeStrategy.scala

```diff
@@ -673,4 +673,26 @@ private[sql] class CarbonLateDecodeStrategy extends SparkStrategy {
     supportCodegen && vectorizedReader.toBoolean &&
     cols.forall(_.dataType.isInstanceOf[AtomicType])
   }
+
+  private def createHadoopFSRelation(relation: LogicalRelation) = {
+    val sparkSession = relation.relation.sqlContext.sparkSession
+    relation.catalogTable match {
+      case Some(catalogTable) =>
+        HadoopFsRelation(new CatalogFileIndex(
```

Comment:
move `new CatalogFileIndex(` to next line

Reply:
If parameter list is long, it is better to add the parameter name also for better readability.

Reply:
Ok, moved down.